openapi.go

```diff
@@ -173,30 +174,67 @@ func RegisterOpenAPIOperation[T, B any](openapi *OpenAPI, route Route[T, B]) (*o
 	}
 
 	route.GenerateDefaultDescription()
-
 	if route.Operation.Summary == "" {
 		route.Operation.Summary = route.NameFromNamespace(camelToHuman)
 	}
 
 	if route.Operation.OperationID == "" {
 		route.GenerateDefaultOperationID()
 	}
-
 	// Request Body
 	if route.Operation.RequestBody == nil {
 		bodyTag := SchemaTagFromType(openapi, *new(B))
-
 		if bodyTag.Name != "unknown-interface" {
 			requestBody := newRequestBody[B](bodyTag, route.RequestContentTypes)
-
-			// add request body to operation
 			route.Operation.RequestBody = &openapi3.RequestBodyRef{
 				Value: requestBody,
 			}
 		}
 	}
 
-	// Response - globals
+	typeOfT := reflect.TypeOf(*new(T))
```

Comment:
It's not `T`, it would be another type to represent Params.

Please put all this code in another generic function `RegisterParams[Params any](params Params, operation openapi3.Operation)`.